assets/airports/kclt.json

```diff
@@ -0,0 +1,7898 @@
+        "TYI"  : ["N35.9767236", "W77.7037317"],
+        "TYMER": ["N34.8104444", "W80.5398361"],
+        "UGANG": ["N34.8068639", "W81.0577972"],
+        "UNARM": ["N34.6549111", "W81.6427139"],
+        "UNJAM": ["N34.2156944", "W80.8030028"],
+        "VALLL": ["N35.0651111", "W81.0551083"],
+        "WALUT": ["N34.6666444", "W80.8421750"],
+        "WAYDS": ["N35.4432000", "W81.2757333"],
+        "WEAZL": ["N35.6738944", "W81.0864667"],
+        "WHDRW": ["N34.8288389", "W81.2517583"],
+        "WHFRD": ["N35.5563528", "W81.0768000"],
+        "WHIZE": ["N35.4240389", "W80.7320750"],
+        "WHYLE": ["N34.9472194", "W81.2663111"],
+        "WILUM": ["N36.4377472", "W81.8099000"],
+        "WLLGO": ["N35.6122056", "W81.6048444"],
+        "WLLSH": ["N35.8892722", "W80.2026806"],
+        "WOOOO": ["N35.3243083", "W80.9318389"],
+        "WRLLD": ["N36.8029000", "W79.6174028"],
+        "WUUTN": ["N35.6221806", "W82.1968028"],
+        "VXV"  : ["N35.9048386", "W83.8947264"],
+        "YELLS": ["N34.6937361", "W80.0494194"],
+        "YNGUN": ["N35.1890639", "W81.5332250"],
+        "ZYMKO": ["N34.8453250", "W80.5739833"]
+    },
+    "runways": [
+        {
+            "name": ["5", "23"],
+            "end": [
+                ["N35.2089524", "W80.9499457", "705.3ft"],
+                ["N35.2226162", "W80.9311454", "746.9ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["18L", "36R"],
+            "end": [
+                ["N35.2247354", "W80.9361570", "746.6ft"],
+                ["N35.2009571", "W80.9341172", "723.9ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["18C", "36C"],
+            "end": [
+                ["N35.2273964", "W80.9531693", "742.2ft"],
+                ["N35.1999923", "W80.9508116", "692.2ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["18R", "36L"],
+            "end": [
+                ["N35.2252828", "W80.9674186", "744.4ft"],
+                ["N35.2006188", "W80.9652964", "743.4ft"]
+            ],
+            "ils": [true, true]
+        }
+    ],
+    "sids": {
+        "BARMY4": {
+                    "icao": "BARMY4",
+                    "name": "Barmy Four",
+                    "rwy": {
+                        "KCLT05": ["#055"],
+                        "KCLT18L": [["_RWY18L1DME", "A12+"], "#183"],
+                        "KCLT18C": [["_RWY18C1DME", "A12+"], "#183"],
+                        "KCLT18R": [["_RWY18R1DME", "A12+"], "#200"],
+                        "KCLT23": ["#235"],
+                        "KCLT36L": [["_RWY36L1DME", "A12+"], "#315"],
+                        "KCLT36C": [["_RWY36C1DME", "A12+"], "#330"],
+                        "KCLT36R": [["_RWY36R1DME", "A12+"], "#025"]
+                    },
+                    "body": ["MESHY", "GULFY", "BARMY", "BATTA"],
+                    "exitPoints": {
```

Comment:
L381: BARMY is also a valid exit point (applies to other SIDs as well)

Reply:
Fixed